src/test/java/com/training/api/tests/ga/GuestAccountTests.java

```diff
@@ -250,4 +250,195 @@ public void testCreateGuestAccount(){
                 .as("Account id pattern doesn't match")
                 .matches("^[{]?[0-9a-fA-F]{8}-([0-9a-fA-F]{4}-){3}[0-9a-fA-F]{12}[}]?$");
     }
+
+
+    @Test
+    public void testNegativeGaCreationWrongAppKey(){
+        String errorCode= "COMMONS-0001";
+
+        headerMap.put(map.get("gaAppKeyHeaderName"),
+                RandomStringUtils.randomAlphanumeric(map.get("gaAppKeyHeaderValue").length()));
+
+        String firstName="Audrey";
+        String lastName="Poole";
+        String email= String.format("%s%s@email.com",
+                RandomStringUtils.randomAlphabetic(5),
+                System.currentTimeMillis());
+
+        com.training.pojos.ga.creation.Request request = new com.training.pojos.ga.creation.Request();
+        request.setBirthdate("19620802");
+        request.setEmail(email);
+        request.setFirstName(firstName);
+        request.setLastName(lastName);
+        request.setMarketingCountry("USA");
+        request.setPassword("Password1");
+        request.setPrivacyAcceptDateTime("20190524T090712GMT");
+        request.setPrivacyVersion("1.11");
+        request.setTncAcceptDateTime("20190524T090712GMT");
+        request.setTncVersion("1.8");
+        request.setSecurityQuestion("What was the first concert you attended?");
+        request.setSecurityQuestionKey("WHAT_WAS_THE_FIRST_CONCERT_YOU_ATTENDED");
+        request.setSecurityAnswer("Answer1");
+        request.setUidType("EMAIL");
+
+        response = new RestEngine().getResponse(map.get("gaCreateUrl"),headerMap,
+                new Gson().toJson(request));
+
+        com.training.pojos.ga.creation.Response responseElement =
+                response.as(com.training.pojos.ga.creation.Response.class);
+
+        Assertions.assertEquals(401,responseElement.getStatus(),REQUEST_UNSUCCESSFUL);
+
+        softAssertions.assertThat(responseElement.getErrors().get(0).getInternalMessage().length()).as(ERRORS_MESSAGE)
+                .isGreaterThan(0);
+
+        softAssertions.assertThat(responseElement.getErrors().get(0).getErrorCode()).as(ERROR_CODE)
+                .isEqualTo(errorCode);
+        softAssertions.assertThat(responseElement .getError().getErrorCode()).as(ERROR_CODE)
+                .isEqualTo(errorCode);
+        softAssertions.assertThat(responseElement.getError().getMessage().length()).as(ERRORS_MESSAGE)
+                .isGreaterThan(0);
```

Comment:
instead validating the length of the error message, lets validate the actual error message here.